pkgs/development/tools/build-managers/cmake/default.nix

```
@@ -8,6 +8,7 @@
 , useOpenSSL ? !isBootstrap, openssl
 , useNcurses ? false, ncurses
 , withQt5 ? false, qtbase
+, useSphinx ? (!isBootstrap && (useNcurses || withQt5)), sphinx, texinfo
```

```suggestion
, buildManPages ? (!isBootstrap && (useNcurses || withQt5)), sphinx, texinfo
```

It doesn't matter with which tool they are build. The important part is that they are build IMO.

---

Fair point, but I'll call the parameter `buildDocs`, because it builds man pages as well as info pages.